catalog/transformer/catalog-transformer-streaming-lib/src/main/java/org/codice/ddf/transformer/xml/streaming/lib/XmlInputTransformer.java

```diff
@@ -259,7 +248,7 @@ public MetacardType getMetacardType() {
      * @return a DynamicMetacardType that describes the type of metacard that is created in this transformer
      */
 
-    public DynamicMetacardType defineMetacardType() {
+    public MetacardType getMetacardType() {
         Set<AttributeDescriptor> attributeDescriptors = new HashSet<>();
```

Comment:
We don't know the size before hand - not without looping through all the factories at least (And then we would loop through again to populate the set). Those two are just the base attributes that will be included on every metacard type